Dockerfile

```diff
@@ -1,8 +1,14 @@
 FROM node:16-alpine3.15
 
-ENV NODE_ENV production
+RUN apk add --no-cache ca-certificates \
+ && apk upgrade --no-cache \
+ && addgroup -S app \
+ && adduser -S app -G app -u 31337 -h /app/ \
+ && chown -R app:app /app/
 
+USER app
 WORKDIR /app
+ENV NODE_ENV production
 
 COPY --chown=node:node package.json package-lock.json /app/
```

Comment:
Need to change the user from node to app on these 2 lines

Reply:
Done